projects/fal/tests/test_stability.py

```diff
@@ -557,10 +557,10 @@ def get_env_var(name: str) -> str | None:
 
 
 def test_fal_storage(isolated_client):
+    url_prefix = "https://fal-cdn.fly.dev/files"
```

Comment:
Wanted to parametrize and test it out, but turns out pickling is not happy about running into monkeypatch'ed STORAGE_TYPE. Gave up for now.